docs/user-guide/amor/amor-reduction.ipynb

```diff
@@ -170,12 +189,34 @@
    "outputs": [],
    "source": [
     "from ess.reflectometry.tools import scale_reflectivity_curves_to_overlap\n",
-    "results_scaled = dict(zip(\n",
-    "    results.keys(),\n",
-    "    scale_reflectivity_curves_to_overlap(results.values()),\n",
-    "    strict=True\n",
-    "))\n",
-    "sc.plot(results_scaled, norm='log', vmin=1e-5)"
+    "\n",
+    "scaled_reflectivity_curves = scale_reflectivity_curves_to_overlap(reflectivity.values())\n",
```

Comment:
I wouldn't call that an assumption. That is explicitly the interface of the function. From the docs: `Make the curves overlap by scaling all except the first by a factor.`

I don't think always using the curve with minimum Q as the baseline  (the curve that the others are scaled relative to) is a good idea. That just makes the routine cumbersome to use in case you want the baseline to be a curve with a minimum Q higher than the lowest Q among all curves.

If you think the current interface is too implicit, maybe it would be better to have a signature like this:
```python
scale_curves_to_overlap(curves, relative_to=baseline_curve)
```

Reply:
Yes I think a more explicit interface would be better. That said, it's sort of unrelated to this PR, so we should change it in another PR.